docs/topics/native/native-binary-options.md

```diff
@@ -0,0 +1,301 @@
+                <li><code>false</code> (default)</li>
+            </list>
+        </td>
+        <td>Controls support for Latin-1-encoded strings to reduce application binary size and adjust memory consumption.</td>
+        <td>Experimental since 2.2.0</td>
+    </tr>
+    <tr>
+        <td><a href="native-memory-manager.md" anchor="track-memory-consumption-on-apple-platforms"><code>mmapTag</code></a></td>
+        <td><code>UInt</code></td>
+        <td>Controls memory tagging, necessary for memory consumption tracking on Apple platforms. Values <code>240</code>-<code>255</code> are available (default is <code>246</code>); <code>0</code> disables tagging</td>
+        <td>Available since 2.2.0</td>
+    </tr>
+    <tr>
+        <td><code>disableMmap</code></td>
+        <td>
+            <list>
+                <li><code>true</code></li>
+                <li><code>false</code> (default)</li>
+            </list>
+        </td>
+        <td>Controls the default allocator. When <code>true</code>, uses the <code>malloc</code> memory allocator instead of <code>mmap</code>.</td>
+        <td>Available since 2.2.0</td>
+    </tr>
+    <tr>
+        <td><code>gc</code></td>
+        <td>
+            <list>
+                <li><a href="native-memory-manager.md" anchor="optimize-gc-performance"><code>cms</code></a></li>
+                <li><a href="native-memory-manager.md" anchor="disable-garbage-collection"><code>noop</code></a></li>
+                <li><code>stms</code> (default)</li>
+            </list>
+        </td>
+        <td>Controls garbage collection behavior: <code>cms</code> enables concurrent marking to decrease GC pause time, <code>noop</code> disables garbage collection.</td>
+        <td><code>cms</code> is Experimental since 2.0.20</td>
+    </tr>
+    <tr>
+        <td><a href="native-memory-manager.md" anchor="garbage-collector"><code>gcMarkSingleThreaded</code></a></td>
+        <td>
+            <list>
+                <li><code>true</code></li>
+                <li><code>false</code> (default)</li>
+            </list>
+        </td>
+        <td>Disables parallelization of the mark phase in garbage collection. May increase GC pause time on large heaps.</td>
+        <td>Available since 1.7.20</td>
+    </tr>
+    <tr>
+        <td><a href="native-memory-manager.md" anchor="monitor-gc-performance"><code>enableSafepointSignposts</code></a></td>
+        <td>
+            <list>
+                <li><code>true</code></li>
+                <li><code>false</code> (default)</li>
+            </list>
+        </td>
+        <td>Enables tracking GC-related pauses in the project for debugging in Xcode Instruments.</td>
+        <td>Available since 2.0.20</td>
+    </tr>
+    <tr>
+        <td><code>preCodegenInlineThreshold</code></td>
+        <td><code>UInt</code></td>
+        <td>Configures the inlining optimization pass, which comes before the actual code generation phase. The recommended number of tokens is 40.</td>
```

Comment:
> Configures the inlining optimization pass, which comes before the actual code generation phase.

Configures Kotlin IR inline optimization pass. 

Also worth noting that it is disabled by default.